Tools/autotest/common.py

```diff
@@ -2002,81 +1971,26 @@ def test_gripper(self):
                                                                            ))
         self.context_pop()
         self.reboot_sitl()
-    #     # TEST MISSION FILE
-    #     # TODO : rework that to work on autotest server
-    #     # self.progress("TEST LOADING MISSION")
-    #     # num_wp = self.load_mission(
-    #                  os.path.join(testdir, "fake_mission.txt"))
-    #     # if num_wp == 0:
-    #     #     self.progress("Failed to load all_msg_mission")
-    #     #     sucess = False
-    #     #
-    #     # self.progress("TEST SAVING MISSION")
-    #     # num_wp_old = num_wp
-    #     # num_wp = self.save_mission_to_file(os.path.join(testdir,
-    #                                          "fake_mission2.txt"))
-    #     # if num_wp != num_wp_old:
-    #     #     self.progress("Failed to save all_msg_mission")
-    #     #     sucess = False
-    #
-    #     self.progress("TEST CLEARING MISSION")
-    #     self.mavproxy.send("wp clear\n")
-    #     self.mavproxy.send('wp list\n')
-    #     self.mavproxy.expect('Requesting [0-9]+ waypoints')
-    #     num_wp = mavwp.MAVWPLoader().count()
-    #     if num_wp != 0:
-    #         self.progress("Failed to clear mission ")
-    #         sucess = False
-    #
-    #     return sucess
-    #
-    # # TESTS FAILSAFE
-    # @abc.abstractmethod
-    # def test_throttle_failsafe(self, home, distance_min=10, side=60,
-    #                            timeout=180):
-    #     """Test that RTL success in case of thottle failsafe."""
-    #     pass
-    #
-    # # TEST ARM RADIO
-    # @abc.abstractmethod
-    # def test_arm_motors_radio(self):
-    #     """Test arming with RC sticks."""
-    #     pass
-    #
-    # # TEST DISARM RADIO
-    # @abc.abstractmethod
-    # def test_disarm_motors_radio(self):
-    #     """Test disarming with RC sticks."""
-    #     pass
-    #
-    # # TEST AUTO DISARM
-    # @abc.abstractmethod
-    # def test_autodisarm_motors(self):
-    #     """Test auto disarming."""
-    #     pass
-    #
-    # # TEST RC OVERRIDE
-    # # TEST RC OVERRIDE TIMEOUT
-    # @abc.abstractmethod
-    # def test_rtl(self, home, distance_min=10, timeout=250):
-    #     """Test that RTL success."""
-    #     self.progress("# Enter RTL")
-    #     self.mavproxy.send('switch 3\n')
-    #     tstart = self.get_sim_time()
-    #     while self.get_sim_time() < tstart + timeout:
-    #         m = self.mav.recv_match(type='VFR_HUD', blocking=True)
-    #         pos = self.mav.location()
-    #         home_distance = self.get_distance(home, pos)
-    #         self.progress("Alt: %u  HomeDistance: %.0f" %
-    #                        (m.alt, home_distance))
-    #         if m.alt <= 1 and home_distance < distance_min:
-    #             self.progress("RTL Complete")
-    #             return True
-    #     return False
-    #
-    # @abc.abstractmethod
-    # def test_mission(self, filename):
-    #     pass
+
+    def is_copter(self):
```

Comment:
I was imagining that `arducopter.py` would override a base-class implementation which returns `False` to return `True`.

That way this list never gets out of date (which it is at the moment...)